userspace/libsinsp/parsers.cpp

```diff
@@ -5075,9 +5075,9 @@ void sinsp_parser::parse_chroot_exit(sinsp_evt *evt) {
 		// if we happen to change container id
 		//
 		if(container_id != evt->get_tinfo()->m_container_id) {
-			evt->get_tinfo()->set_user(evt->get_tinfo()->m_user.uid());
-			evt->get_tinfo()->set_loginuser(evt->get_tinfo()->m_loginuser.uid());
-			evt->get_tinfo()->set_group(evt->get_tinfo()->m_group.gid());
+			evt->get_tinfo()->set_group(evt->get_tinfo()->m_gid);
+			evt->get_tinfo()->set_user(evt->get_tinfo()->m_uid);
+			evt->get_tinfo()->set_loginuser(evt->get_tinfo()->m_loginuid);
```

Comment:
This line is doing `evt->get_tinfo()->m_loginuid = evt->get_tinfo()->m_loginuid;` or am i wrong?